samples/tlssurvey/build.gradle.kts

```diff
@@ -0,0 +1,26 @@
+plugins {
+  kotlin("jvm")
+  application
+  id("com.google.devtools.ksp").version("1.6.21-1.0.6")
+}
+
+application {
+  mainClass.set("okhttp3.survey.RunSurveyKt")
+}
+
+dependencies {
+  implementation("com.squareup.okhttp3:okhttp:5.0.0-alpha.10")
```

Comment:
Including a dependency on the published OkHttp interferes pretty badly with IntelliJ. I’m gonna send a PR to remove this.

Reply:
https://github.com/square/okhttp/pull/8323